feature-group-definitions/sanitizer.yml

```diff
@@ -1,2 +1,3 @@
 name: Sanitizer
+description: The Sanitizer API sanitizes untrusted strings of HTML, Document and DocumentFragment objects. After sanitization, unwanted elements or attributes are removed, and the returned objects can safely be inserted into a document's DOM.
```

Comment:
This will merge into HTML and may no longer be called Sanitizer API after that. Can it be described in terms of `setHTML()` and `parseHTML()`?